draft-ietf-quic-transport.md

```diff
@@ -5579,8 +5585,8 @@ value.
 An endpoint that receives a NEW_CONNECTION_ID frame with a sequence number
 smaller than the Retire Prior To field of a previously received
 NEW_CONNECTION_ID frame MUST immediately send a corresponding
-RETIRE_CONNECTION_ID frame that retires the newly received connection ID.
-
+RETIRE_CONNECTION_ID frame that retires the newly received connection ID,
+unless it has already done so for that sequence number.
```

Comment:
Was the concern about doing this multiple times, or was it about the `immediately` part? That still wasn't clear based on the discussion on the list.

Reply:
I would also lose the "immediately".  That's not a useful word to use in this context.

Reply:
@erickinnear The multiple times part.
@martinthomson good suggestion; I will patch it after I make dinner.